<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeInfoForm</class>
 <widget class="QWidget" name="NodeInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Node number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nodeNumEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>388</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Eccentricity:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="eccentricityLabel">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Degree:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="degreeLabel">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Shortest loop:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="shortLoopLabel">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Adjacent nodes</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="adjacentNodesList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="resizeMode">
           <enum>QListView::Adjust</enum>
          </property>
          <property name="batchSize">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Projection</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="createProjectionButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Create projection</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="levelTable">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>75</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>70</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>level</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>original</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>replica</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>total</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
